backend/wait_for_db.py

```
@@ -0,0 +1,21 @@
+import time
+import MySQLdb
+from django.db import connections
```

Ruff (0.8.2)

`django.db.connections` imported but unused

Remove unused import: `django.db.connections`

(F401)